backend/src/test/nl/tudelft/ti2806/riverrush/domain/entity/AnimalTest.java

```diff
@@ -59,9 +60,139 @@ public void testDrop() throws Exception {
     }
 
     @Test
-    public void testReturnToBoat() throws Exception {
+     public void testReturnToBoat() throws Exception {
```

Comment:
All these are state transitions and therefore should be tested in the test class of that state